sources/accessible-name-and-description.ts

```diff
@@ -560,7 +560,7 @@ export function computeTextAlternative(
 			recursion: boolean;
 		}
 	): string {
-		if (consultedNodes.has(current)) {
+		if (consultedNodes.has(current) && !context.isReferenced) {
```

Comment:
leftover from earlier iteration
```suggestion
		if (consultedNodes.has(current)) {
```